maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -194,9 +194,6 @@ class AndroidDriver(
         launchArguments: Map<String, Any>,
         sessionId: UUID?,
     ) {
-        if(!open) // pick device flow, no open() invocation
-            open()
```

Comment:
As mentioned, I had to remove 2 lines of code: I have to be honest, I didn't understand their purpose. 
So I'd love some help from @sdfgsdfgd or @igorsmotto  to know why they're needed. 🙏

Here the issue is that since `maestro test` has found an active session, it didn't "open" the Android driver since it doesn't need to. 
But this condition would force it to reopen when using `launchApp` which defeats the purpose of `maestro start-session`